backend/plugin/db/elasticsearch/elasticsearch.go

```diff
@@ -169,20 +169,21 @@ func (d *Driver) QueryConn(_ context.Context, _ *sql.Conn, statement string, _ d
 	for _, s := range statements {
 		if err := func() error {
 			startTime := time.Now()
+			// It's ok for routes without the leading '/' in the editor.
```

Comment:
Instead of handling the missing "/" prefix, shall we handle it in the splitElasticsearchStatements()?

Reply:
That will be more concise, but I originally thought the splitElasticsearchStatements shouldn't handle HTTP standards.